src/app/core/common-components/entity-form/form-field-label/form-field-label.component.html

```diff
@@ -0,0 +1,3 @@
+<span *ngIf="_field" [matTooltip]="_field.description">
+  {{ _field?.label ?? _field?.id }}
```

Comment:
Is this our current approach as well to fall back to the `.id`? I am not sure whether this should be supported or if we rather obviously show that there is something missing.